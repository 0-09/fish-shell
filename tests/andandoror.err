
####################
# Basic && and || support

####################
# && and || in if statements

####################
# && and || in while statements

####################
# Complex scenarios
