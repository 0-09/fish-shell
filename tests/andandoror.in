# Test && and || support

echo first && echo second
echo third || echo fourth
true && false ; echo "true && false: $status"
true || false ; echo "true || false: $status"
true && false || true ; echo "true && false || true: $status"
