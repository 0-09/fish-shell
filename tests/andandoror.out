first
second
third
true && false: 1
true || false: 0
true && false || true: 0
